tooling/lib/updated.js

```diff
@@ -18,7 +18,7 @@ const {diff} = require('./git');
  * npm instead of upstream/master
  * @returns {Promise<Array<string>>}
  */
-exports.updated = async function updated({dependents, npm}) {
+exports.updated = async function updated({dependents, npm = !!process.env.CI}) {
```

Comment:
I'm ok with making this empty instead of defaulted false if there is no value
I just added it cuz it seemed "appropriate"